package.json

```diff
@@ -124,6 +124,10 @@
                 {
                     "command": "vscode-docker.registries.azure.selectSubscriptions",
                     "when": "never"
+                },
+                {
+                    "command": "vscode-docker.help",
+                    "when": "never"
```

Comment:
Any reason *not* to allow the command from the palette?

Reply:
Because Morty said we have a lot of commands already so let's hold off on this for now ☺️